smokeflow.rb

```diff
@@ -0,0 +1,118 @@
+require 'rspec'
+require 'watir'
+
+describe "Automation Practice: " do
```

Comment:
It is better practice to use single quotes: e.g. 'Automation Practice' 
Check: [https://stackoverflow.com/questions/6395288/double-vs-single-quotes](url)